nautobot/extras/models/relationships.py

```diff
@@ -435,10 +435,10 @@ class Relationship(BaseModel, ChangeLoggedModel, NotesMixin):
     objects = RelationshipManager()
 
     class Meta:
-        ordering = ["name"]
+        ordering = ["label"]
 
     def __str__(self):
-        return self.name.replace("_", " ")
+        return self.label.replace("_", " ")
 
```

Comment:
I'd expect to see a `check_if_key_is_graphql_safe` added to the clean() method for Relationship?